src/Sylius/Component/Report/Model/Report.php

```diff
@@ -88,11 +81,9 @@ public function getName()
     }
 
     /**
-     * Sets the value of name.
-     *
-     * @param string $name the name
+     * @param string $name
```

Comment:
So just to clarify, shall I also remove all the `return $this;` statements in setters and make them non-fluent as part of this pull request?